tabbie2.git/api/controllers/RoundController.php

```diff
@@ -46,7 +46,6 @@ public function actionFilter($tournament_id = null)
 			'query' => Round::find()
                 ->where(["round.tournament_id" => $tournament_id])
                 ->andWhere(["round.published" => 1])
-                ->andWhere(["round.displayed" => 1])
```

Comment:
@wanaryytel this is then highly problematic since the motion is a property of the round.
Meaning this will leak the motion to everyone (with an api access - so everyone who signs up) the moment the round is published (but NOT yet displayed) 
... please change back in.

If you want to send data in advance, you can create a new model (e.g preRound that holds that data - no motion etc)
Please revert this change.
cc @rscoates